eeps/eep-0064.md

```diff
@@ -0,0 +1,354 @@
+    Author: Kiko Fernandez-Reyes <kiko(at)erlang(dot)org>,
+        Raimo Niskanen <raimo(at)erlang(dot)org>
+    Status: Draft
+    Type: Standards Track
+    Created: 07-Jun-2023
+    Erlang-Version: OTP-27
+    Post-History:
+****
+EEP 64: Multi-line Indented Verbatim Strings
+----
+
+Abstract
+========
+
+This EEP proposes the introduction of Multi-line Indented Verbatim strings,
+*MIV Strings* and defines their semantics.  The main benefit is to allow
+*multi-line* strings in an easy and useful, *indented*, way,
+similar to in other languages, e.g Elixir.
+
+Their first use case is for in-module documentation attributes
+containing [Markdown][] or similarly formatted text where *verbatim* text
```

Comment:
Is `[Markdown][]` expected? It doesn't seem to be valid Markdown.

Reply:
It is. Just `[Markdown]` works too but it might be a GitHub-Flavoured Markdown thing.

Reply:
[Markdown][] < `[Markdown][]`
[Markdown] < `[Markdown]`

^ doesn't seem to do anything in particular in GitHub-Flavoured Markdown.

Reply:
I don't know what the above means but please not that in order for `[ref]` and `[ref][]` to link, you need a `[ref]: url` somewhere in your Markdown.

Reply:
> I don't know what the above means

It was to show you (to the left) how stuff was rendered, but José's reply enlightened me. I've used this pattern before, mostly in `README.md`s, but somehow this just fell odd seeing it today 😄 Must be Friday-tiredness...